src/uni_modules/wot-ui/components/wd-select-picker/wd-select-picker.vue

```
@@ -337,10 +337,24 @@ function getFilterText(label: string, filterVal: string): Array<{ type: 'active'
 
 function handleFilterChange({ value }: { value: string }) {
   filterVal.value = value
-  if (value === '') {
-    filterColumns.value = props.columns
+  if (props.filterHandler) {
+    callInterceptor(props.filterHandler, {
+      args: [props.columns, value, value], done: () => {
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)